src/Text/Pretty/Simple.hs

```diff
@@ -389,7 +389,7 @@ pHPrintStringNoColor = pHPrintStringOpt NoCheckColorTty defaultOutputOptionsNoCo
 -- | Like 'pShow', but without color.
 --
 -- >>> pShowNoColor [ Nothing, Just (1, "hello") ]
--- "[ Nothing\n, Just\n    ( 1\n    , \"hello\"\n    )\n]"
+-- "[ Nothing\n, Just \n    ( 1\n    , \"hello\"\n    )\n]"
```

Comment:
Do we care about this? I personally don't. I think there are probably other cases where we have some trailing whitespace but our tests don't catch this when we use `pPrint` and similar rather than showing the string.

Reply:
Ok, looking at the history, it seems this was explicitly implemented in https://github.com/cdepillabout/pretty-simple/pull/62. I'll fix it now.